.github/CODEOWNERS

```diff
@@ -4,7 +4,9 @@
 /tests/unit/modin/ @snowflakedb/snowpandas
 /docs/modin_api_coverage/ @snowflakedb/snowpandas
 /docs/source/modin/ @snowflakedb/snowpandas
+/tests/notebooks/ @snowflakedb/snowpandas
 /.github/ @snowflakedb/snowpandas @snowflakedb/snowpark-python-api-reviewers
+/docs/ @snowflakedb/snowpandas @snowflakedb/snowpark-python-api-reviewers
```

Comment:
Shouldn't need this, should default to Snowpark Python for all other directories
```suggestion
```